molecule/resources/tasks/create_network.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Create network in gcp project, a subnet and a firewall rule.
+
+- name: "Create network {{ molecule_yml.driver.network_name }}"
```

Comment:
I'm unclear on why we're creating/deleting a network on each run. The default network in stackdriver-kubernetes-1337 should have these ports open already.

I'm not really against it per se, it just seems unnecessary. Could you explain why you have this logic?

If you don't want to rely on the firewall rules in the default network, we could consider a hardcoded ansible-test network that has the firewall rules we need.